### PR TITLE
realization cognito aws sdk

### PR DESCRIPTION
**Pull request by edii**

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -14,8 +14,10 @@ import (
 )
 
 func NewClient(basePath string) (*HTTPClient, error) {
-	if basePath == "" {
-		return nil, errors.New("APIBase are required to create a Client")
+	// check AuthServerURL
+	authURL, err := url.Parse(basePath)
+	if err != nil || authURL.Hostname() == "" {
+		return nil, fmt.Errorf("AUTH_SERVER_URL mast contain valid url")
```
Comment (Troush) [inline pkg/client/client.go]: Must

File: services/srv.go [hunk 2]
```diff
@@ -165,3 +172,75 @@ func (s *Service) CognitoUserInfo(req *http.Request) (*ConsumerID, error) {
...
+	}
+
+	if s.CognitoClient == nil {
+		s.tracer.ExtStatus(s.tracer.GetParentSpan(), http.StatusInternalServerError)
+		return nil, ErrInternalServerError
+	}
+
+	// check used context
+	if s.cfg.IsAWSContext() {
+		user, err = s.CognitoClient.GetUserWithContext(context.TODO(), &cognito.GetUserInput{AccessToken: aws.String(splitHeader[1])})
```
Comment (Troush) [inline services/srv.go]: Todo context should be changed

File: pkg/client/client.go (hunk 1 above)
Comment (edii) [reply pkg/client/client.go]: Done

File: services/srv.go (hunk 2 above)
Comment (edii) [reply services/srv.go]: Done

**Pull request merged**